src/odemis/dataio/tiff.py

```diff
@@ -2743,7 +2743,8 @@ def _convert_thermo_fisher_to_odemis_metadata(metadata: str) -> dict:
     try:
         # beam metadata
         beam_type = tfs_md["Beam"]["beam"]
-        md[model.MD_ROTATION] = float(tfs_md[beam_type]["scanrotation"])
+        # tmp: disable scan rotation metadata, as it is inconsistent with how odemis displays images
+        md[model.MD_ROTATION] = 0  # float(tfs_md[beam_type]["scanrotation"])
```

Comment:
If it is consistently flipped by 180 degrees, why not just add 180 degrees to `float(tfs_md[beam_type]["scanrotation"])`?

Reply:
To be more specific, the display rotation should depend on both scan rotation and stage rotation. But stage rotation = 0 isnt the flat to sem rotation, depends on the specific microscope. So we'd need to read the configuration metadata (which isn't really something we'd want to depend on at image loading time, because it might be different (e.g. different microscope)). So for now its just easier to disable the rotation, so that users see the same images as they took

Reply:
Yes, I've seen similar issues before. The problem is that the way the SEM image should be displayed depends a lot on the rest of the system. Sometimes we really want to see it rotated, sometime it should be shown at 0°, because the scan rotation is there precisely to compensate for some other rotation! 
I think it's fine to "play dumb" and just not report the scan rotation. That's how it's shown in the SEM software anyway.